snippets/python/src/closed_channel.py

```diff
@@ -0,0 +1,20 @@
+import breez_sdk
+def prepare_sweep(fee_rate):
+    # ANCHOR: prepare_sweep
```

Comment:
nit: for consistency
```suggestion
    # ANCHOR: prepare-sweep
```

Reply:
Agree, Done